packages/amplify-graphql-default-value-transformer/src/__tests__/amplify-grapphql-default-value-transformer.test.ts

```diff
@@ -338,4 +198,90 @@ describe('DefaultValueModelTransformer:', () => {
     expect(out.resolvers['Mutation.createNote.init.1.req.vtl']).toBeDefined();
     expect(out.resolvers['Mutation.createNote.init.2.req.vtl']).toBeUndefined();
   });
+
+  it.each([{ strategy: mockSqlDataSourceStrategy() }])(
+    'throws if auto-increment is implied on a non-Postgres datasource',
+    ({ strategy }) => {
+      const schema = `
+      type CoffeeQueue @model {
+        id: ID! @primaryKey
+        orderNumber: Int! @default
+        name: String
+      }`;
+      expect(() => {
+        testTransform({
+          schema: schema,
+          transformers: [new ModelTransformer(), new DefaultValueTransformer(), new PrimaryKeyTransformer()],
+          dataSourceStrategies: constructDataSourceStrategies(schema, strategy),
+        });
+      }).toThrow('The @default directive requires a value property on non Postgres datasources.');
+    },
+  );
+
+  it.each([
+    { typeStr: 'Boolean' },
+    { typeStr: 'AWSJSON' },
+    { typeStr: 'AWSDate' },
+    { typeStr: 'AWSDateTime' },
+    { typeStr: 'AWSTime' },
+    { typeStr: 'AWSTime' },
+    { typeStr: 'AWSURL' },
+    { typeStr: 'AWSPhone' },
+    { typeStr: 'AWSIPAddress' },
+  ])('throws if auto-increment is implied on non-int types', ({ typeStr }) => {
```

Comment:
Can you confirm that the test output includes the `typeStr` value in the test log? If a test fails, we want to know which value is problematic.

Reply:
added `typeStr` to the logged message:
```
'throws if @default (no args) is used on non-int types (case: $typeStr)'
```